packages/@guardian/source-foundations/src/typography/typography.stories.mdx

```diff
@@ -61,48 +74,39 @@ typography API assigns font size in rems.
 font-family: 'GH Guardian Headline, Guardian Egyptian Web, Georgia, serif';
 ```
 
--   `headline.xxxsmall()` -> 17px
--   `headline.xxsmall()` -> 20px
--   `headline.xsmall()` -> 24px
--   `headline.small()` -> 28px
--   `headline.medium()` -> 34px
--   `headline.large()` -> 42px
--   `headline.xlarge()` -> 50px
+<Story name="headline">
+	<FontStylesRenderer fontName="headline" fontStyles={headlineObjectStyles} />
+</Story>
 
 ### Body
 
 ```css
 font-family: 'GuardianTextEgyptian, Guardian Text Egyptian Web, Georgia, serif';
 ```
 
--   `body.xsmall()` -> 14px
--   `body.small()` -> 15px
--   `body.medium()` -> 17px
+<Story name="body">
```

Comment:
do these need to be a story? aren't they now stories within a story?

Reply:
The reason for using <Story /> tags is that anything outside of that tag is not picked up by Chromatic in an .mdx story, regardless of what we set as the <Meta /> tag.

If we make each individual story { viewMode: 'canvas' } then we can click into them as standard in Storybook, but we then can't view the docs by clicking on typography in the sidebar.

Reply:
This thread is relevant: https://github.com/storybookjs/storybook/issues/9209